numba/runtime/nrt.c

```diff
@@ -208,7 +208,7 @@ static
 void nrt_internal_dtor(void *ptr, void *info) {
     NRT_Debug(nrt_debug_print("nrt_internal_dtor %p, %p\n", ptr, info));
     if (info != NULL) {
-        memset(ptr, 0, (size_t)info);  /* for safety */
+        memset(ptr, 0xDE, (size_t)info);  /* for safety */
```

Comment:
Yes. Zeroing could actually hide bugs, especially in the zeros() implementation :)